### PR TITLE
Address TODOs in the installation guides

### PR DESCRIPTION
**Pull request by sabbyanandan**
Address TODOs where possible in the installation guides; comment the items that aren't required for MVP (2 of them still remains).

Replace all bit.ly references with dataflow.spring.io URLs.

Update subtle UX corrections.

### CODE REVIEW

File: content/documentation/pages/1-installation/2-cloudfoundry/2-cf-local.md [hunk 1]
```diff
@@ -42,14 +40,23 @@ Now you are ready to start the server application, as follows:
 
     java -jar spring-cloud-dataflow-server-{%dataflow-version%}.jar --spring.config.additional-location=<PATH-TO-FILE>/foo.properties
```
Comment (cppwfs) [inline content/documentation/pages/1-installation/2-cloudfoundry/2-cf-local.md]: We can remove the { } from the java command

File: content/documentation/pages/1-installation/1-local/3-manual.md [hunk 2]
```diff
@@ -52,20 +54,16 @@ These instructions require that RabbitMQ be running on the same machine as Skipp
...
         java -jar spring-cloud-dataflow-server-{%dataflow-version%}.jar --spring.cloud.skipper.client.serverUri=https://192.51.100.1:7577/api
 
-    If you want to use the shell to use Data Flow, start it with the following command:
-
-        java -jar spring-cloud-dataflow-shell-{%dataflow-version%}.jar
+3.  If you want to use the shell to use Data Flow, start it with the following command:
 
-If the Data Flow Server and shell are not running on the same host, you can also point the shell to the Data Flow server URL by using the
-`dataflow config server` command when in the shell’s interactive mode.
+         java -jar spring-cloud-dataflow-shell-{%dataflow-version%}.jar
```
Comment (cppwfs) [inline content/documentation/pages/1-installation/1-local/3-manual.md]: we can remove the `{` `}`

File: content/documentation/pages/1-installation/1-local/3-manual.md [hunk 3]
```diff
@@ -10,28 +10,30 @@ If Docker does not suit your needs, you can manually install the parts you need
 
 ## Download server jars
 
-1.  Download the Spring Cloud Data Flow Server and shell by using the following command:
+1.  Download the Spring Cloud Data Flow Server and shell by using the following commands:
 
         wget https://repo.spring.io/{version-type-lowercase}/org/springframework/cloud/spring-cloud-dataflow-server/{%dataflow-version%}/spring-cloud-dataflow-server-{%dataflow-version%}.jar
```
Comment (cppwfs) [inline content/documentation/pages/1-installation/1-local/3-manual.md]: %dataflow-version% is not resolving to version number

File: content/documentation/pages/1-installation/1-local/3-manual.md [hunk 4]
```diff
@@ -10,28 +10,30 @@ If Docker does not suit your needs, you can manually install the parts you need
...
 
         wget https://repo.spring.io/{version-type-lowercase}/org/springframework/cloud/spring-cloud-dataflow-shell/{%dataflow-version%}/spring-cloud-dataflow-shell-{%dataflow-version%}.jar
 
-2.  Download [Skipper](https://cloud.spring.io/spring-cloud-skipper/) (because
-    Data Flow delegates to Skipper for those features), by running the
+2.  Download [Skipper](https://cloud.spring.io/spring-cloud-skipper/) by running the
     following commands:
 
-        wget https://repo.spring.io/{skipper-version-type-lowercase}/org/springframework/cloud/spring-cloud-skipper-server/%skipper-version%/spring-cloud-skipper-server-%skipper-version%.jar
+        wget https://repo.spring.io/%skipper-version%/org/springframework/cloud/spring-cloud-skipper-server/%skipper-version%/spring-cloud-skipper-server-%skipper-version%.jar
```
Comment (cppwfs) [inline content/documentation/pages/1-installation/1-local/3-manual.md]: %xyz-version% is not getting replaced in generated doc.

**Comment by cppwfs** [comment]
Scanned through the doc real quick and put up my comments.

File: content/documentation/pages/1-installation/1-local/3-manual.md (hunk 3 above)
Comment (sabbyanandan) [reply content/documentation/pages/1-installation/1-local/3-manual.md]: Yeah, I brought it up with @oodamien on a slack thread earlier today. It seems like a bug.

**Comment by cppwfs** [comment]
There are a few more http links in the doc.  I'll correct them on merge.

**Comment by cppwfs** [comment]
Fixing Maven Configuration link in cf-cli doc on merge.  It was returning 404.

**Comment by sabbyanandan** [comment]
Happy to redo any bits. Just let me know.

**Comment by cppwfs** [comment]
LGTM, Polished, Squashed,Merged

**Pull request closed without merging**